graphics/src/Pbr_TEST.cc

```diff
@@ -28,10 +29,14 @@ TEST(Pbr, BasicAPI)
   EXPECT_EQ(gz::common::PbrType::NONE, pbr.Type());
   EXPECT_EQ(std::string(), pbr.AlbedoMap());
   EXPECT_EQ(std::string(), pbr.NormalMap());
+  EXPECT_EQ(nullptr, pbr.NormalMapData());
   EXPECT_EQ(gz::common::NormalMapSpace::TANGENT, pbr.NormalMapType());
   EXPECT_EQ(std::string(), pbr.RoughnessMap());
+  EXPECT_EQ(nullptr, pbr.RoughnessMapData());
   EXPECT_EQ(std::string(), pbr.MetalnessMap());
+  EXPECT_EQ(nullptr, pbr.MetalnessMapData());
   EXPECT_EQ(std::string(), pbr.EmissiveMap());
+  EXPECT_EQ(nullptr, pbr.EmissiveMapData());
```

Comment:
I didn't go too deep in editing all the tests for the PBR object, it still has a lot of test code for testing all the copy, assignment, copy-assignment, move operators etc. that are a leftover from when the class was not using the `gz-utils` implementation pointer and all the operators were implemented manually, i.e. [here](https://github.com/gazebosim/gz-common/pull/163/files#diff-60e733fab278ef8fb8a50a97bd9270374f2cf2cc67b2bafad190eb0ea0e47f46L60-L80). I believe all the tests are now unnecessary and can be removed but that's out of the scope of this (already too wide!) PR.